plugins/frontend-app/.toolkitrc.yml

```diff
@@ -1,6 +1,6 @@
 plugins:
```

Comment:
**question**: do we want to rename this plugin too? right now it would be weird to have a frontend app not hosted on Heroku, but i could see that happening in the future with e.g. Compute@Edge